components/map/overview.md

```diff
@@ -56,16 +56,14 @@ Tha layers are responsible for organizing the Map information. [Read more about
 
 You can add different points with real coordinates on the map as markers. [Read more about the Blazor Map Markers...]({%slug components/map/layers/marker%})
 
-## Zoom and Navigation
+## Pan and Zoom
 
 Raster maps are divided into images (tiles) for serving over the web. Tiles are typically 256px squares. The top level (zoom level 0) displays the whole world as a single tile. Each progressive zoom level doubles the size of the Map. You can set the zoom level through the `Zoom` property. The zoom operation can be performed with a double click on the map or by using the mouse scroll wheel.
```

Comment:
The order of the sentences can be reversed - first say that zooming & panning operations are possible (and how) and then add details what happens with tile layers. Adding a sentence about shape layers might also be useful as it will not be clear if zoom & pan is supported.